### PR TITLE
add alpha-3 and numeric country codes and unit test for it

### LINKED ISSUE #529: Adapt CountryCode Rule to accept alpha-3 and number

### PR DESCRIPTION
**Pull request by fefas**
fix #529

### CODE REVIEW

File: library/Rules/CountryCode.php [hunk 1]
```diff
@@ -1,41 +1,318 @@
...
 
-/*
- * This file is part of Respect/Validation.
- *
- * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
- *
- * For the full copyright and license information, please view the "LICENSE.md"
- * file that was distributed with this source code.
- */
-
```
Comment (henriquemoody) [inline library/Rules/CountryCode.php]: This header is required, I'm sorry for not saying this before.

Comment (henriquemoody) [inline library/Rules/CountryCode.php]: Any way, don't worry about it, I will fix it here.

Comment (fefas) [inline library/Rules/CountryCode.php]: I copied from [this file](https://github.com/Respect/ValidationLocale/blob/master/Rules/Country.php) eheh.

No problem, I will fix it.

**Comment by henriquemoody** [comment]
Merged manually by 141103f085a44558b42d8f2c1bfd4420a4ee29f6

Thanks for contributing! :panda_face:

**Pull request closed without merging**